### DISCUSSION TITLE
Splat benchmark

### DISCUSSION THREAD

**Discussion by mysteryx93** [DevTeam/Pure.DI#23] [General]
Could you add [Splat](https://github.com/reactiveui/splat) to the benchmark? Requires [Splat.DI.SourceGenerator](https://github.com/reactiveui/Splat.DI.SourceGenerator)

Pure.DI and Splat actually look pretty similar, I'm curious as to the difference between the two.

Since both are source generators, performance should be pretty close. Any feature that one has over the other?

**Comment by NikolayPianikov** [comment]
@mysteryx93 Yes, the ideas are really the same. I'll [do it](https://github.com/DevTeam/Pure.DI/issues/24)!
